### DISCUSSION TITLE
Radio calibration doubt

### DISCUSSION THREAD

**Discussion by fsclavo** [egzumer/uv-k5-firmware-custom#134] [Q&A] [answered]
Hi! I'm just learning about radio calibration data (and the recomendation to back it up). I've flashed fagci and egzumer firmwares to my radio to try them both, using egzumer uvtools (https://egzumer.github.io/uvtools/): my question is if there is any possibility that I've overwritten calibration data without noticing it. If not (hope so), I'll take a backup with k5prog as recommended.
Thanks!

**Comment by egzumer** [comment, marked as answer]
This is experimental software, it is provided as is and there are no guarantees. I know of one case where a software bug caused the calibration data to be erased. So there is that risk. Having said that...

In general, we (FW creators) don't overwrite anything in the calibration section of the EEPROM without user input. For example, in my FW there is an option to calibrate the battery, so if you do a battery calibration then yes 2 bytes will be changed in the calibration data.

Otherwise the FW shouldn't change anything and you should be fine. Having a backup of the calibration data is always better than not having it, but in theory it should never be needed.

**Reply by fsclavo** [reply]
I've calibrated battery with your firmware, therefore indicator is working correctly now, so I'm happy with this change. I'll take a dump of that just in case.
Thanks for the detailied answer and for your firmware!
